test/e2e_node/image_list.go

```diff
@@ -61,11 +61,11 @@ var NodeImageWhiteList = sets.NewString(
 	"gcr.io/kubernetes-e2e-test-images/node-perf/tf-wide-deep-amd64:1.0",
 )
 
-// updateImageWhiteList updates the framework.ImageWhiteList with
+// updateImageAllowList updates the framework.ImageWhiteList with
```

Comment:
we should also update the `framework.ImageWhiteList` variable?

Reply:
Done. I wanted to do it in smaller pieces to have this one as an example but might as well rename ImageWhiteList from the framework now.

Reply:
given how painful it is to get PRs in currently, and how tiny this PR is, I think it might make more sense to combine these.

we should probably name these variables and methods consistently anyhow?

Reply:
ah the other 2 issues for node and common images are already being worked on.

on consistent naming: yes I think so. i'm looking at https://github.com/kubernetes/kubernetes/pull/92323 and it seems to be that the image "white lists" are indeed lists of images to prepull so yes to consistent naming.